internal/support/logging/router_test.go

```diff
@@ -295,7 +307,7 @@ func TestRemoveLogs(t *testing.T) {
 			matchCriteria{LogLevels: logLevels, OriginServices: services, Start: 1, End: 2}},
 	}
 	// create test server with handler
-	ts := httptest.NewServer(LoadRestRoutes())
+	ts := httptest.NewServer(LoadRestRoutes(mockDIC()))
```

Comment:
See my other comment regarding simplifying this test to target the REST handler rather than loading all the routes and making a mock HTTP request.

Reply:
See above.